crates/router/src/connector/expresscheckout/transformers.rs

```diff
@@ -0,0 +1,363 @@
+    Charged,
+    Success,
+    Created,
+    PendingVbv,
+    AuthenticationFailed,
+    #[default]
+    AuthorizationFailed,
+    JuspayDeclined,
+    CaptureInitiated,
+    CaptureFailed,
+    VoidInitiated,
+    VoidFailed,
+    Voided,
+    Authorized,
+    Authorizing,
+    PendingAuthentication,
+}
+
+impl From<ExpresscheckoutPaymentStatus> for enums::AttemptStatus {
+    fn from(item: ExpresscheckoutPaymentStatus) -> Self {
+        match item {
+            ExpresscheckoutPaymentStatus::Charged => Self::Charged,
+            ExpresscheckoutPaymentStatus::Success => Self::Charged,
+            ExpresscheckoutPaymentStatus::Created => Self::Started,
+            ExpresscheckoutPaymentStatus::AuthenticationFailed => Self::AuthenticationFailed,
+            ExpresscheckoutPaymentStatus::AuthorizationFailed => Self::AuthorizationFailed,
+            ExpresscheckoutPaymentStatus::Authorizing => Self::Authorizing,
+            ExpresscheckoutPaymentStatus::Authorized => Self::Authorized,
+            ExpresscheckoutPaymentStatus::PendingVbv => Self::AuthenticationPending,
+            ExpresscheckoutPaymentStatus::JuspayDeclined => Self::Failure,
+            ExpresscheckoutPaymentStatus::CaptureFailed => Self::CaptureFailed,
+            ExpresscheckoutPaymentStatus::CaptureInitiated => Self::CaptureInitiated,
+            ExpresscheckoutPaymentStatus::VoidInitiated => Self::VoidInitiated,
+            ExpresscheckoutPaymentStatus::VoidFailed => Self::VoidFailed,
+            ExpresscheckoutPaymentStatus::Voided => Self::Voided,
+            ExpresscheckoutPaymentStatus::PendingAuthentication => Self::AuthenticationPending,
+        }
+    }
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+pub struct ExpresscheckoutPaymentsResponse {
+    #[serde(default, deserialize_with = "deserialize_error_default")]
+    status: ExpresscheckoutPaymentStatus,
+    txn_uuid: Option<String>,
+    payment: Option<Authentication>,
+}
+
+fn deserialize_error_default<'de, D, ExpresscheckoutPaymentStatus>(
+    deserializer: D,
+) -> Result<ExpresscheckoutPaymentStatus, D::Error>
+where
+    ExpresscheckoutPaymentStatus: Default + Deserialize<'de>,
+    D: Deserializer<'de>,
+{
+    let opt = ExpresscheckoutPaymentStatus::deserialize(deserializer);
+    match opt {
+        Ok(v) => Ok(v),
+        _ => Ok(ExpresscheckoutPaymentStatus::default()),
+    }
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+pub struct Authentication {
+    authentication: AuthenticationData,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+pub struct AuthenticationData {
+    url: String,
+    method: Method,
+}
+
+impl<F, T>
+    TryFrom<
+        types::ResponseRouterData<
+            F,
+            ExpresscheckoutPaymentsResponse,
+            T,
+            types::PaymentsResponseData,
+        >,
+    > for types::RouterData<F, T, types::PaymentsResponseData>
+{
+    type Error = error_stack::Report<errors::ParsingError>;
```

Comment:
Don't use `ParsingError` inside the connector code, It should be `ConnectorError`. 

This should be generated by the add_connector.sh script we will fix that from our side.